Unity.gitignore

```diff
@@ -26,6 +26,9 @@
 # Visual Studio cache directory
 .vs/
 
+# Visual Studio Code cache directory
+.vscode/
```

Comment:
We have VS code settings and extensions recommendations commited and not ignored. I wonder what the recommandation is regarding `.vscode`.